### PR TITLE
Task/#106 screen videos

### LINKED ISSUE #106: Screen - Videos
imagem
https://app.zeplin.io/project/5e711e6973c497b6352ed9f3/screen/5e73b66e22a4ddb1c4c83416

imagem
https://app.zeplin.io/project/5e711e6973c497b6352ed9f3/screen/5e73b66e1285030494b00dee

imagem
https://app.zeplin.io/project/5e711e6973c497b6352ed9f3/screen/5e711e8b3e6a6c01f8aa4e77

### PR DESCRIPTION
**Pull request by Vanethos**
closes #106 

**Description**
* Adds Video Page
* Adds Video Player Page
* Adds getVideoId to model
* Fixes Video Player Widget

**Screenshots**
WhatsApp Image 2020-03-22 at 12 38 30(1)

WhatsApp Image 2020-03-22 at 12 38 30

### CODE REVIEW

File: lib/ui/screens/video_player/video_player_page.dart [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+class _VideoPlayerPageState extends State<VideoPlayerPage> {
+  YoutubePlayerController _controller;
+  String _url;
+
+  @override
+  Widget build(BuildContext context) {
+    _url = ModalRoute.of(context).settings.arguments;
+      _controller = YoutubePlayerController(
```
Comment (miguelpruivo) [inline lib/ui/screens/video_player/video_player_page.dart]: I'd probably have this assigned on `initState`

File: lib/ui/screens/video_player/video_player_page.dart [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        initialVideoId: _url,
+        flags: YoutubePlayerFlags(
+          autoPlay: true,
+        ),
+      );
+    return Scaffold(
+      appBar: AppBar(
+        backgroundColor: Colors.black,
+        elevation: 0,
```
Comment (miguelpruivo) [inline lib/ui/screens/video_player/video_player_page.dart]: You can remove `elevation` from here, #91 will include a theme property for it.

File: lib/ui/screens/video_player/video_player_page.dart [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import 'package:covid19mobile/ui/app.dart';
```
Comment (danidroid) [inline lib/ui/screens/video_player/video_player_page.dart]: @Vanethos licence is missing

File: lib/ui/screens/videos/videos_page.dart [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    ///
+    /// If pushing from home and faqs have initial data
+    /// In case of no initial data reverts to fetch faqs
+    /// and update with the Provider
+    _videos = Provider.of<VideosProvider>(context).videos ??
+        ModalRoute.of(context).settings.arguments;
+
+    return Scaffold(
+      appBar: AppBar(
+        iconTheme:
```
Comment (miguelpruivo) [inline lib/ui/screens/videos/videos_page.dart]: I would probably remove all properties from `AppBar` here, as they are being set on theme with #91.

File: lib/ui/screens/video_player/video_player_page.dart (hunk 3 above)
Comment (Vanethos) [reply lib/ui/screens/video_player/video_player_page.dart]: Will add it

File: lib/ui/screens/video_player/video_player_page.dart (hunk 1 above)
Comment (Vanethos) [reply lib/ui/screens/video_player/video_player_page.dart]: cannot be in `initState`due to the plugin

**Pull request merged**